docs/sphinx/api/default_ops.rst

```diff
@@ -638,31 +638,67 @@ operations, each operating on 2 qubits.
 Photonic Operations on Qudits
 =============================
 
-These operations are valid only on the `photonics` target which does not support the quantum operations above.
+These operations are valid only on the `photonics-cpu` target which does not support
+the quantum operations above.
 
-:code:`plus`
+:code:`create`
```

Comment:
Q: Is this a new gate? or an alias for 'plus'? Doesn't match the examples and API spec...

Reply:
This is a new gate replacing the `plus` that can be deprecated, the behaviour of the `plus` gate is not very intuitive (cycling through the number of photons).